examples/step-55/doc/intro.dox

```diff
@@ -0,0 +1,141 @@
+
+Since $P$ is aimed to be a preconditioner only, we shall use
+approximations to the inverse of the Schur complement $S$ and the
+matrix $A$.  Therefore, in the above equations, $-M_p=\hat{S} \approx
+S$, where $M_p$ is the pressure mass matrix and is solved by using CG
++ ILU, and $\hat{A}$ is an approximation of $A$ obtained by one of
+multiple methods: CG + ILU, just using ILU, CG + GMG (Geometric
+Multigrid as described in Step-16), or just performing a few V-cycles
+of GMG. The inclusion of CG is more expensive, in general.
+
+As a comparison, instead of FGMRES, we also use the direct solver
+UMFPACK to compare our results to.  If you want to use UMFPACK as a
+solver, it is important to note that since you have a singular system
+(since the integral of mean pressure being equal to zero not
+implemented), we set the first pressure node equal to zero since the
+direct solver can not handle the singular system like the other
+methods could.
+
+<h3> Reference Solution </h3>
+
+The domain, right hand side, and boundary conditions we implemented
+were chosen for their simplicity and the fact that they made it
+possible for us to compute errors using a reference solution.  We
+apply Dirichlet boundary condtions for the whole velocity on the whole
+boundary of the domain Ω=[0,1]×[0,1]×[0,1].  To enforce the boundary
+conditions we can just use our reference solution that we will now
+define.
+
+Let $u=(u_1,u_2,u_3)=(2\sin (\pi x), - \pi y \cos (\pi x),- \pi z \cos
+(\pi x))$ and $p = \sin (\pi x)\cos (\pi y)\sin (\pi z)$.
+
+If you look up in the deal.ii manual what is needed to create a class
+inherited from <code>Function@<dim@></code>, you will find not only a
+value function, but vector_value, value_list, etc.  Different things
+you use in your code will require one of these particular
+functions. This can be confusing at first, but luckily the only thing
+you actually need to implement is value.  The other ones have default
+implementations inside deal.ii and will be called on their own as long
+as you implement value correctly.
+
+Notice that our reference solution fulfills $\nabla \cdot u = 0$. In
+addition, the pressure is chosen to have a mean value of zero.  For
+the Method of Manufactured Solutions of Step-7, we need to find $\bf
+f$ such that:
+
+@f{align*}
+{\bf f} =   - 2 \text{div} \frac {1}{2} \left[ (\nabla \textbf{u}) + (\nabla \textbf{u})^T\right] + \nabla p. 
+@f}
+
+Using the reference solution above, we obtain:
+
+@f{eqnarray*} 
+{\bf f} &=& (2 \pi^2 \sin (\pi x),- \pi^3 y \cos(\pi
+x),- \pi^3 z \cos(\pi x))\\ & & + (\pi \cos(\pi x) \cos(\pi y)
+\sin(\pi z) ,- \pi \sin(\pi y) \sin(\pi x) \sin(\pi z), \pi \cos(\pi
+z) \sin(\pi x) \cos(\pi y)) @f}
+
+<h3> Computing Errors </h3> 
+Because we do not enforce the mean
+pressure to be zero for our numerical solution in the linear system,
+we need to postprocess the solution after solving. To do this we use
+the <code>compute_mean_value</code> function to compute the mean value
```

Comment:
I think you use `VectorTools::compute_mean_value()`. If so, just use the fully qualified name and doxygen will automatically link to the right function. (No `<code>` needed in that case.)